lib/core/utils/flattened-tree.js

```diff
@@ -47,9 +68,7 @@ function virtualDOMfromNode(node, shadowId) {
 			return vNodeCache._tabbableElements;
 		}
 	};
-	if (axe._nodeMap) {
-		axe._nodeMap.set(node, vNode);
-	}
+	axe._cahce.nodeMap.set(node, vNode);
```

Comment:
ahem

Reply:
Also outdated and corrected in another comit